src/uct/ib/base/ib_iface.h

```diff
@@ -357,4 +359,66 @@ void uct_ib_iface_set_max_iov(uct_ib_iface_t *iface, size_t max_iov)
 }
 
 
+static UCS_F_ALWAYS_INLINE
+void uct_ib_iface_fill_ah_attr_raw(
+        uct_ib_iface_t *iface, uint16_t lid,
+        union ibv_gid *gid, uint8_t *path_bits_p,
```

Comment:
1. gid can be const*
1. better pass just 'uint8_t path_bits'

Reply:
gid marked as const.

path_bits here is pointer to value, like workaround for optional argument (actually separate path_bits value is used for wireup only), and non-NULL value of pointer means that should be used non-iface path_bits value